src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java

```diff
@@ -28,53 +28,44 @@
 package org.llorllale.cactoos.matchers;
 
 import org.hamcrest.Description;
-import org.hamcrest.MatcherAssert;
 import org.hamcrest.StringDescription;
 import org.hamcrest.core.IsEqual;
 import org.junit.Test;
 
 /**
- * Test case for {@link NotBlank}.
+ * Test case for {@link IsBlank}.
  *
  * @since 1.0.0
  * @checkstyle JavadocMethodCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-public final class NotBlankTest {
+public final class IsBlankTest {
 
     @Test
     public void blank() {
-        MatcherAssert.assertThat(
-            "must not match an empty string",
-            new NotBlank().matchesSafely(
-                "", new Description.NullDescription()
-            ),
-            new IsEqual<>(false)
-        );
+        new Assertion<>(
+        "must match an empty string",
+            () -> "",
+            new IsBlank()
+        ).affirm();
     }
 
     @Test
     public void notBlank() {
-        MatcherAssert.assertThat(
-            "must match a non-empty string",
-            new NotBlank().matchesSafely(
-                "-.$%", new Description.NullDescription()
-            ),
-            new IsEqual<>(true)
+        new Assertion<>(
+        "must not match a non-empty string",
+            () -> "-.$%",
+            new IsBlank()
         );
     }
 
     @Test
     public void nonBlankMessage() {
         final Description desc = new StringDescription();
-        MatcherAssert.assertThat(
-            "must match a non-empty string",
-            new NotBlank().matchesSafely("text", desc),
-            new IsEqual<>(true)
-        );
-        MatcherAssert.assertThat(
-            "must describe itself in terms of the text being matched against",
-            desc.toString(),
+        new IsBlank().matchesSafely("text", desc);
```

Comment:
Use 
````java
@Rule
public ExpectedException 
````
in order to check the negative scenario as described here
https://github.com/llorllale/cactoos-matchers/blob/master/src/test/java/org/llorllale/cactoos/matchers/ScalarHasValueTest.java#L75

The using of `matchesSafely` was a bad idea and now we have the separate task to fix all tests regarding it.